Update the tests to work with the latest jupyter server

This was noticed when iterating on https://github.com/voila-dashboards/voila-gridstack/pull/69

We should fix the tests so they pass with the latest releases of Jupyter Server. And remove this pin added in https://github.com/voila-dashboards/voila-gridstack/pull/69:

https://github.com/voila-dashboards/voila-gridstack/blob/27a8402a2a5d9b678034af345615fd75c08d80df/setup.py#L116

Some of the changes include the rename of the fixtures, that now include the `jp` prefix. 

An example run where the tests fail: https://github.com/voila-dashboards/voila-gridstack/runs/1557959257

With:

```
E       fixture 'base_url' not found
```

![image](https://user-images.githubusercontent.com/591645/102396321-9760f180-3fdc-11eb-819b-88c84043c6d7.png)

--- a/setup.py
+++ b/setup.py
@@ -9,7 +9,6 @@ from jupyter_packaging import (
     combine_commands, skip_if_exists
 )
 import setuptools
-from setuptools.command.develop import develop
 
 try:
     import jupyter_core.paths as jupyter_core_paths
@@ -112,13 +111,12 @@ setup_args = dict(
     cmdclass=cmdclass,
     packages=setuptools.find_packages(),
     install_requires=[
-        "jupyterlab_widgets>=1.0.0rc1",
+        "jupyterlab_widgets~=1.0",
         "voila>=0.2.0,<0.3.0"
     ],
     extras_require={
         "test": [
             'ipykernel',
-            'jupyter_server~=1.0.1',
             'pytest',
             'pytest-tornasync',
             'lxml'

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -22,6 +22,11 @@ def voila_app(voila_args, voila_config):
 
 
 @pytest.fixture
+def base_url():
+    return "/"
+
+
+@pytest.fixture
 def app(voila_app):
     return voila_app.app
 

